Packetery/Checkout/Model/ResourceModel/Order/CollectionFactory.php

```diff
@@ -45,4 +45,13 @@ public function create(array $data = [])
         $collection->joinSalesOrder();
         return $collection;
     }
+
+    public function createForDbInsert(array $data = [])
```

Comment:
Protože je to specifická metoda, asi bych přidal i PHPDoc s vysvětlením té metody.